scss/_patterns_icons.scss

```diff
@@ -89,6 +89,10 @@
       vertical-align: 0;
     }
   }
+
```

Comment:
The styles above this one (for icons inline with headings) define `u-match--h<heading_level>` classes, can we get a `.u-match-p` for this for consistency?

Reply:
TBH, I'm not exactly sure what's the purpose of all those `u-match-X` class names, they don't seem to be defined or used anywhere else. We probably don't have to remove them now, but also I don't think it's necessary to add more.

Reply:
@bartaz As far as I can tell, it is only used (internally) by [MAAS UI](https://github.com/canonical/maas-ui/blob/ee87049981cfc9edce97539deb58bccc729e6791/src/app/zones/views/ZonesList/ZonesListHeader/ZonesListTitle/ZonesListTitle.tsx#L9). ([search ref](https://github.com/search?q=org%3Acanonical+u-match-&type=code))

So, we can probably ignore this